components/avatar/avatar.html

```diff
@@ -0,0 +1,195 @@
+              ></path>
+            </g>
+          </svg>
+        </nav>
+      </header>
+
+      <div class="main-body">
+        <aside class="main-sidebar">
+          <span class="sidebar-header">Components</span>
+          <ul class="sidebar-items">
+            <a href="../alert/alert.html" class="sidebar-item">Alert</a>
+            <a href="#" class="sidebar-item active">Avatar</a>
+            <a href="../badge/badge.html" class="sidebar-item">Badge</a>
+            <a href="../button/button.html" class="sidebar-item">Button</a>
+            <a href="../card/card.html" class="sidebar-item">Card</a>
+            <a href="../image/image.html" class="sidebar-item">Image</a>
+            <a href="../input/input.html" class="sidebar-item">Input</a>
+          </ul>
+        </aside>
+        <div class="main-content-container">
+          <div class="main-content">
+            <h1 class="component-title">Avatar</h1>
+            <p class="component-info">
+              The Avatar component is used to represent a user, and displays the
+              profile picture, initials or fallback icon.
+            </p>
+            <div class="component-type">Simple Avatar</div>
+
+            <div class="component-example">
+              <div class="avatar-container">
+                <img
+                  class="avatar md"
+                  src="https://bit.ly/dan-abramov"
+                  alt="medium logo"
+                />
+                <img
+                  class="avatar md"
+                  src="https://bit.ly/dan-abramov"
+                  alt="medium logo"
+                />
+                <img
+                  class="avatar md"
+                  src="https://bit.ly/dan-abramov"
+                  alt="medium logo"
+                />
+                <img
+                  class="avatar md"
+                  src="https://bit.ly/dan-abramov"
+                  alt="medium logo"
+                />
+                <img
+                  class="avatar md"
+                  src="https://bit.ly/dan-abramov"
+                  alt="medium logo"
+                />
+                <img
+                  class="avatar md"
+                  src="https://bit.ly/dan-abramov"
+                  alt="medium logo"
+                />
```

Comment:
In my opinion, even one avatar would be enough here rather than adding 6 avatars of the same class.